### DISCUSSION TITLE
Scroll behavior while output is generated.

### DISCUSSION THREAD

**Discussion by philipp-lehmann** [janhq/jan#4260] [Feature Ideas]
I recently started using Jan.ai and I am really thrilled about this project. 
So far I haven't found this issue, but please apologize if I missed something.

While text output is created the UI is always jumping, scrolling down. I often scroll upwards to read the start of the answer or older answers, but this gets really hard, because more and more text is added.

https://github.com/user-attachments/assets/4892f34b-015b-4d9f-b304-7e6eff4ace43

Suggestion: 
- Stop scrolling once a user scroll has been performed.
- Add smoother scrolling while generating. ie timeout 450ms, scroll down with ease-in-out in 150ms. 
- Make the timing adjustable in the options.

**Comment by tenten71** [comment]
I agree. This is an issue with other tools as well. The chat window should not scroll at all. My recommendation is that it should process the response with a "Response is generating" alert that switches to "Response has completed". The alert can auto vanish when I click anywhere in the window. But it should stay until I do so in case I walked away from my computer.
An additional way to indicate what's happening is to colorize the scroll bar while it is "generating" the response and then return to normal color when it finishes.
I'm curious to know if this auto scrolling causes accessibility issues.
